src/elements/content-sidebar/SidebarNav.scss

```diff
@@ -63,3 +63,88 @@
         }
     }
 }
+
+.is-responsive-web {
+    @include breakpoint($small-screen) {
+        .bcs .bcs-NavButton,
+        .bcs-SidebarNav-secondary .btn-plain,
+        .bdl-AdditionalTab {
+            width: 46px;
+            margin-right: 0;
+        }
+    }
+
+    @include breakpoint($medium-screen) {
+        .bcs-SidebarNav,
+        .bcs-SidebarNav-tabs {
+            flex-direction: row;
+        }
+
+        .bcs-SidebarNav-main {
+            display: flex;
+            flex-direction: row;
+        }
+
+        .bcs-NavButton::before {
```

Comment:
Can we group this with and move after the other `.bcs-NavButton` styles? Can we also move those styles to the SidebarNavButton component stylesheet?

Reply:
Consolidated `.bcs-NavButton` styles, and moved it to `SidebarNavButton.scss`